src/TestData/TestData-2017.csproj

```diff
@@ -0,0 +1,303 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFrameworks>net35;netstandard1.6</TargetFrameworks>
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+        <AssemblyName>MathNet.Numerics.TestData</AssemblyName>
+        <RootNamespace>MathNet.Numerics.TestData</RootNamespace>
+    </PropertyGroup>
+    <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+        <OutputPath>..\..\out\test\Net35\</OutputPath>
+        <IntermediateOutputPath>..\..\obj\test\Net35\</IntermediateOutputPath>
+        <BaseIntermediateOutputPath>..\..\obj\test\Net35\</BaseIntermediateOutputPath>
+        <DebugType>pdbonly</DebugType>
+        <Optimize>true</Optimize>
+        <DefineConstants>TRACE</DefineConstants>
+        <ErrorReport>prompt</ErrorReport>
+        <WarningLevel>4</WarningLevel>
+        <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
+        <NoWarn>1591</NoWarn>
+        <Prefer32Bit>false</Prefer32Bit>
+        <LangVersion>6</LangVersion>
+    </PropertyGroup>
+    <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+        <DefineConstants>DEBUG;TRACE</DefineConstants>
+        <OutputPath>..\..\out\test-debug\Net35\</OutputPath>
+        <IntermediateOutputPath>..\..\obj\test-debug\Net35\</IntermediateOutputPath>
+        <BaseIntermediateOutputPath>..\..\obj\test-debug\Net35\</BaseIntermediateOutputPath>
+        <DebugSymbols>true</DebugSymbols>
+        <DebugType>full</DebugType>
+        <Optimize>false</Optimize>
+        <ErrorReport>prompt</ErrorReport>
+        <WarningLevel>4</WarningLevel>
+        <PlatformTarget>AnyCPU</PlatformTarget>
+        <NoWarn>1591</NoWarn>
+        <Prefer32Bit>false</Prefer32Bit>
+        <LangVersion>6</LangVersion>
+    </PropertyGroup>
+    <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release-Signed|AnyCPU'">
+        <OutputPath>..\..\out\test-signed\Net35\</OutputPath>
+        <IntermediateOutputPath>..\..\obj\test-signed\Net35\</IntermediateOutputPath>
+        <BaseIntermediateOutputPath>..\..\obj\test-signed\Net35\</BaseIntermediateOutputPath>
+        <!-- Conditional Strong Name: YES -->
+        <SignAssembly>true</SignAssembly>
+        <DebugType>pdbonly</DebugType>
+        <Optimize>true</Optimize>
+        <DefineConstants>TRACE;STRONGNAME</DefineConstants>
+        <ErrorReport>prompt</ErrorReport>
+        <WarningLevel>4</WarningLevel>
+        <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
+        <NoWarn>1591</NoWarn>
+        <Prefer32Bit>false</Prefer32Bit>
+        <LangVersion>6</LangVersion>
+    </PropertyGroup>
+    <ItemGroup>
+        <Reference Include="System" />
+        <Reference Include="System.Core" />
+        <Reference Include="System.Xml.Linq" />
+        <Reference Include="System.Data.DataSetExtensions" />
+        <Reference Include="System.Data" />
+        <Reference Include="System.Xml" />
+    </ItemGroup>
```

Comment:
I believe you are correct. I've made the changes and everything's still working fine. Will wait for any additional feedback from Christoph before pushing.